content/docs/api-reference/make_checkpoint.md

```diff
@@ -115,15 +102,8 @@ Experiment results have been applied to your workspace.
 > DVC checkpoints to behave as expected.
```

Comment:
There may be other errors too, so i am not sure whether we should show example of catching `KeyboardInterrupt`.

Reply:
I think the code sample was taken from an actual checkpoints example repo which is a demo meant to run continuously until you Ctrl + C.

Reply:
See https://github.com/iterative/dvc-checkpoints-mnist/pull/2

Reply:
@dberenbaum, I think that could be and should be explained separately. I'd keep it simple for `every 100th iteration` example.